tests/functional/adapter/utils/test_utils.py

```diff
@@ -23,6 +24,10 @@
     seeds__data_hash_csv,
     models__test_hash_sql,
     models__test_hash_yml,
+    models__test_generate_series_sql,
```

Comment:
Are these files from dbt core?

Reply:
Hi @SatishChGit ,
dbt-core also provides these but they have it very generic and that didn't work with teradata. So I modified them and added them to the utils.fixtures_utils files and then imported them here to use in testcases.